packages/protocol/contracts/rrp/requesters/RrpBeaconServer.sol

```diff
@@ -99,6 +99,7 @@ contract RrpBeaconServer is
         bytes calldata data
     ) external override onlyAirnodeRrp {
         bytes32 templateId = requestIdToTemplateId[requestId];
+        require(templateId != bytes32(0), "templateId is zero");
```

Comment:
So I've added the test that does do that but because of the way `.call` works it reverts with `Fulfillment failed`  rather than `request ID unknown`. Also due to the way `onlyAirnodeRrp` works there is no other way to call this function except a call from with AirnodeRrp.
 It still reverts at the exact condition though, I verified it by making the mapping public and console.log(requestIdToTemplateId[requestId])

Reply:
Ah okay makes sense. I think it's fine, especially if coverage is 100%